common/lib/xmodule/xmodule/css/sequence/display.scss

```diff
@@ -239,47 +239,20 @@ $sequence--border-color: #C8C8C8;
 
 .sequence-nav-button {
   @extend %ui-depth3;
+  @include transition(all .2s $ease-in-out-quad 0s);
   position: absolute;
   display: block;
   top: 0;
-  width: ($baseline*2);
```

Comment:
If I maintain button width, I'll need to slide the icon over like I had before. You ok with that?